analytic_engine/src/instance/flush_compaction.rs

```diff
@@ -979,16 +975,12 @@ impl SpaceStore {
                 path: sst_file_path.to_string(),
             })?;
 
-        // update sst metadata by built info.
-        sst_meta.row_num = sst_info.row_num as u64;
-        sst_meta.size = sst_info.file_size as u64;
-
         table_data
             .metrics
-            .compaction_observe_output_sst_size(sst_meta.size);
+            .compaction_observe_output_sst_size(sst_info.file_size as u64);
         table_data
             .metrics
-            .compaction_observe_output_sst_row_num(sst_meta.row_num);
+            .compaction_observe_output_sst_row_num(sst_info.row_num as u64);
```

Comment:
How about
```
let sst_file_size  = sst_info.file_size as u64;
let sst_row_num = sst_info.row_num as u64;
```
And reuse it here and follows?

Reply:
fixed